# origin: marcelo (BitKeeper)
# cset: 1.1558 (2.4) key=41e2c4fetTJmVti-Xxql21xXjfbpag
# inclusion: upstream
# descrition: Brad Spengler: Fix random poolsize sysctl (from 2.6.10-ac)
# revision date: Thu, 13 Jan 2005 15:14:00 +0900
#
# S rset: ChangeSet|1.1557..1.1558
# I rset: drivers/char/random.c|1.20..1.21
#
# Key:
# S: Skipped  ChangeSet file only
# O: Original Followed by Updated
# U: Updated  Included with updated range of versions
# I: Included Included verbatim
# E: Excluded Excluded on request from user
# D: Deleted  Manually deleted by subsequent user edit
#
#
# This is a BitKeeper generated diff -Nru style patch.
#
# ChangeSet
#   2005/01/10 16:10:06-02:00 marcelo@logos.cnet 
#   Brad Spengler: Fix random poolsize sysctl (from 2.6.10-ac)
# 
# drivers/char/random.c
#   2005/01/10 16:07:55-02:00 marcelo@logos.cnet +1 -1
#   Brad Spengler: Fix random poolsize sysctl (from 2.6.10-ac)
# 
#
===== drivers/char/random.c 1.20 vs 1.21 =====
--- 1.20/drivers/char/random.c	2004-08-10 08:09:10 +09:00
+++ 1.21/drivers/char/random.c	2005-01-11 03:07:55 +09:00
@@ -1771,7 +1771,7 @@
 static int proc_do_poolsize(ctl_table *table, int write, struct file *filp,
 			    void *buffer, size_t *lenp)
 {
-	int	ret;
+	unsigned int	ret;
 
 	sysctl_poolsize = random_state->poolinfo.POOLBYTES;
 
